content/contents.lr

```diff
@@ -13,11 +13,11 @@ The directory is a curated list of places providing a JSON file based on the [Sc
 A standardized JSON file that can be used to expose information about hackerspaces, makerspaces, fablabs, chaostreffs and the like across the world to web apps or any other application.
 The specification uses [JSON Schema](https://json-schema.org/).
 
-The following example shows how it looks like:
+The following example shows how it looks like (version 14):
```

Comment:
Do we want to have an example which is compatible with both v0.13 and v14? Because that is what people should try to do right now IMHO.

Reply:
Ah, that's a good point :slightly_smiling_face:

Reply:
PR updated.